demo/Makefile

```diff
@@ -3,7 +3,7 @@ VPATH = ../
 OBJECTS = example_als_mcmc.o example_sgd example_sgd_bpr
 INCLUDE = -I../include -L../bin -I../externals/CXSparse/Include -I../externals/OpenBLAS
 CFLAGS = -std=c99 -g -Wall -O0 $(INCLUDE)
-LDLIBS = -L../bin -lfastfm -L../externals/CXSparse/Lib -lcxsparse -L../externals/OpenBLAS -lopenblas -lpthread -lm
+LDLIBS = -L../bin -lfastfm -lopenblas -lpthread -lm
```

Comment:
-lopenblas was added mistakenly. Add -lthread was a solution to make it build on your test machines (on mines it is building without it).  Some open blas references were undefined without it. I need more time to investigate how exactly does it affect openblas.